dipy/denoise/noise_estimate.py

```diff
@@ -136,7 +139,8 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5,
     return sigma
 
 
-def _piesno_3D(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5,
+@deprecated_params("l", "ell", since="1.9", until="2.0")
+def _piesno_3D(data, N, alpha=0.01, ell=100, itermax=100, eps=1e-5,
```

Comment:
Since you are updating this can you pick a more self-explanatory variable name ?

Reply:
Done in e635078.